### PR TITLE
Added voice_chat to eval list

### PR DESCRIPTION
**Pull request by MrHemlock**
Signed-off-by: Daniel Brown <browndj3@gmail.com>

### CODE REVIEW

File: config-default.yml [hunk 1]
```diff
@@ -195,6 +195,7 @@ guild:
         # Voice
         code_help_voice:                    755154969761677312
         code_help_voice_2:                  766330079135268884
+        voice-chat:                         412357430186344448
```
Comment (kwzrd) [inline config-default.yml:198]:
```suggestion
        voice_chat:                         412357430186344448
```

**Review by kwzrd — CHANGES_REQUESTED** [review]

**Comment by Akarys42** [comment]
I think we should allow eval in the whole `VOICE` category instead of manually allowing every channel.

**Comment by MrHemlock** [comment]
> I think we should allow eval in the whole `VOICE` category instead of manually allowing every channel.

Fair point, changing now.

**Review by jerbob — APPROVED** [review]
LGTM.

**Review by kwzrd — APPROVED** [review]

**Pull request merged**